appinfo/routes.php

```diff
@@ -27,5 +27,8 @@
 		['name' => 'settings#getOrder', 'url' => '/getOrder', 'verb' => 'GET'],
 		['name' => 'settings#savePersonal', 'url' => '/savePersonal', 'verb' => 'POST'],
 		['name' => 'settings#saveDefaultOrder', 'url' => '/saveDefaultOrder', 'verb' => 'POST'],
+		['name' => 'settings#getHidden', 'url' => '/getHidden', 'verb' => 'GET'],
```

Comment:
Let's just return two arrays, hidden and order in the getOrder method otherwise there will be 2 http requests.